.github/workflows/integrate.yaml

```diff
@@ -1,106 +1,79 @@
 name: CI
 
 on:
-  - push
-  - pull_request
+  push:
+    branches:
+      - main
+  pull_request:
 
 jobs:
   lint:
-    name: Lint Check
+    name: Lint
     runs-on: ubuntu-latest
 
     steps:
     - name: Check out code
       uses: actions/checkout@v2
-
     - name: Install dependencies
-      run: sudo apt install python3-pip tox
-
+      run: |
+        sudo apt update
```

Comment:
This is pretty small so it's not a big deal, but in general if we're using multi-line scripts, we should be setting `set -eux` at the top. It's much easier to see what exactly failed when the script fails.